lib/config/optionsconfiguration.go

```diff
@@ -184,15 +186,22 @@ func (opts OptionsConfiguration) StunServers() []string {
 	for _, addr := range opts.RawStunServers {
 		switch addr {
 		case "default":
-			defaultPrimaryAddresses := make([]string, len(DefaultPrimaryStunServers))
-			copy(defaultPrimaryAddresses, DefaultPrimaryStunServers)
-			rand.Shuffle(defaultPrimaryAddresses)
-			addresses = append(addresses, defaultPrimaryAddresses...)
+			_, records, err := net.LookupSRV("stun", "udp", "syncthing.net")
+			if err != nil {
+				l.Warnln("Unable to resolve primary STUN servers via DNS:", err)
+			}
+
+			for _, record := range records {
+				var priority = record.Priority
+				var target, _ = strings.CutSuffix(record.Target, ".")
```

Comment:
TrimSuffix or TrimRight are simpler here

Reply:
Oh and prefer `target :=` and `priority :=` instead of `var foo =` syntax, for style consistency reasons